pkg/apis/networking/types.go

```diff
@@ -77,6 +87,26 @@ type NetworkPolicyIngressRule struct {
 	From []NetworkPolicyPeer
 }
 
+// NetworkPolicyEgressRule describes a particular set of traffic that pods can egress to
+// matched by a NetworkPolicySpec's podSelector. The traffic must match both ports and to.
+type NetworkPolicyEgressRule struct {
+	// List of destination ports for outgoing traffic.
+	//Each item in this list is combined using a logical OR. If this field is
+	// empty or missing, this rule matches all ports (traffic not restricted by port).
+	// If this field is present and contains at least one item, then this rule does note
+	// constrain outgoing traffic.
+	// +optional
+	Ports []NetworkPolicyPort
+
+	// List of sources for outgoing traffic of pods selected for this rule.
+	// Items in this list are combined using a logical OR operation. If this field is
+	// empty or missing, this rule matches all sources (traffic not restricted by
+	// source). If this field is present and contains at least on item, this rule
+	// allows traffic only if the traffic matches at least one item in the from list.
+	// +optional
+	To []NetworkPolicyPeer
```

Comment:
// List of ~~sources~~ **destinations** for outgoing traffic of pods selected for this rule.
// Items in this list are combined using a logical OR operation. If this field is
// empty or missing, this rule matches all ~~sources~~ **destinations** (traffic not restricted by
// ~~source~~ **destination**). If this field is present and contains at least ~~on~~ **one** item, this rule
// allows traffic only if the traffic matches at least one item in the ~~from~~ **to** list.

Reply:
`in the to list.` -> `in the To list.`